src/worker/clientcontrol.js

```diff
@@ -432,20 +434,52 @@ commands.SETPASS = async function(input, con, msg) {
 };
 
 commands.ADDTOKEN = async function(input, con, msg) {
+    const parts = input.split(' ');
+    const duration = parts[0] === '0' ? 0 : ParseDuration(parts[0], 'sec');
```

Comment:
Any reason not to leave the 'sec' part to the input? That would make it easier to use for the user if they wanted 1w 1month etc

Reply:
the 'sec' is the returned time saves /1000 the input can be things like `1w1d` or `1y-50w` https://www.npmjs.com/package/parse-duration